src/test_runner.d

```diff
@@ -85,9 +90,4 @@ shared static this()
 
 void main()
 {
-    version(D_Coverage)
-    {
-        import core.runtime : dmd_coverSetMerge;
-        dmd_coverSetMerge(true);
```

Comment:
Well, this has always been wrong anyhow, b/c unittests are run from the runtime initialization before entering main.